src_assets/common/assets/web/scripts/main.js

```diff
@@ -0,0 +1,70 @@
+
```

Comment:
Can you rename this directory to `js`?

Probably no blank line at start of file? Or add a comment describing the purpose of the script?

Reply:
Will do!